src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
Yep, but on with trajectory we don't need to do anything, since native `preventDefault` and `stopPropagation` for focus/blur event are noop. So nothing to see here.

In compat we have simulated bubble `change` and `submit`, but we can add guard for `isSimulated`  property in our `stopPropagation` method (didn't test that).

> We used a plain object before, but it seems to me that nothing we add would be a truly accurate stub, so we may as well delete it.

Not sure why we would delete it, property explicitly called `originalEvent` so it seems putting there original event is right way to do it.

Why not be consistent and add more bytes to this?